app/views/tasks/edit.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="task-container">
+
+<div class="tittle"><h1>HI!! Let's <%= @title %> task!</h1></div>
+<div class="form">
+<%= form_for @task, class: "main-form" do |f| %>
+
+<%= f.label :name, value="New Task" %>
+<%= f.text_field :name, :require => true %>
+<br>
+<%= f.label :description %>
+<%= f.text_field :description %>
+<br>
+
+<%= f.label :completed_at, value="Completed?" %>
+<%= f.text_field :completed_at %>
+<br>
+
+<%= f.collection_select :person_id, Person.all, :id, :name, :require => true, class: "drop" %>
+<br>
```

Comment:
Great use of a drop down! As long as there are not too many People, as I user I personally find drop downs really helpful & useful